https://bugs.gentoo.org/504472

fix building w/readline-6.3 -- the old CPPFunction define has been deleted

--- a/libmisc/ui.c
+++ b/libmisc/ui.c
@@ -256,8 +256,7 @@ static void _init(void) {
 	
 #if defined(HAVE_LIBREADLINE) && defined(HAVE_READLINE_READLINE_H)
 	rl_initialize();
-	rl_attempted_completion_function = 
-		(CPPFunction *)misc_complete;
+	rl_attempted_completion_function = misc_complete;
 #endif
     
 	aal_exception_set_handler(misc_exception_handler);
